### PR TITLE
[0.9.0] Set build context default to Dockerfile dir

### PR DESCRIPTION
**Pull request by anukul**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [x] Other (please describe): Spec change

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] If it's a backend change, tests for the changes have been added and `go test ./...` runs successfully from the root folder.
- [ ] If it's a frontend change, Prettier has been run
- [ ] Docs have been reviewed and added / updated if needed

## What is the current behavior?

Docker build context default is set to repository root

## What is the new behavior?

Docker build context default is set to directory containing Dockerfile

## Technical Spec/Implementation Notes

TODO: Allow setting build context from dashboard

### CODE REVIEW

File: cli/cmd/create.go [hunk 1]
```diff
@@ -223,6 +227,10 @@ func handleSubdomainCreate(subdomain string, err error) error {
 }
 
 func createFromGithub(createAgent *deploy.CreateAgent, overrideValues map[string]interface{}) error {
+	if localPath == "" {
```
Comment (abelanger5) [inline cli/cmd/create.go:230]: Is `createFull` the only method that calls `createFromGithub`? If so, we can remove this right?

**Review by abelanger5 — CHANGES_REQUESTED** [review]
Looks good @anukul, just one minor thing to resolve 👍

File: cli/cmd/create.go (hunk 1 above)
Comment (anukul) [reply cli/cmd/create.go]: That's right.. fixed, thanks.

**Comment by igalakhov** [comment]
Hi @anukul - seems like everything is working well! One slight problem is that (as mentioned in issue #1186) the auto buildpack doesn't work for folders for some reason. I get this error specifically:
> Error: invalid app path 'src/': evaluate symlink: lstat src: no such file or directory
After some investigation, it seems like this happens due to Docker - I suspect something with symlinks is done to build and the way we pull/store the repository breaks things (note: I tested with absolute paths and that has the same issue). The exact line of code that fails is this one (this is in library code):
```go
if resolvedAppPath, err = filepath.EvalSymlinks(appPath); err != nil {
		return "", errors.Wrap(err, "evaluate symlink")
	}
```

**Pull request merged**